src/Chirp.Razor/Pages/UserTimeline.cshtml.cs

```diff
@@ -18,6 +24,12 @@ public UserTimelineModel(ICheepService service)
     public ActionResult OnGet(string author)
     {
         Cheeps = _service.GetCheepsFromAuthor(author);
+        PageNumber = int.TryParse(QueryPage, out var page) && 0 <= page && page <= MathUtil.PageAmount(Cheeps) ? page : 1;
```

Comment:
Instead of repeating the code, maybe make a method to do this?

Reply:
Great idea, I will do that!